keras/preprocessing/image.py

```diff
@@ -792,6 +793,10 @@ def __iter__(self):
     def __next__(self, *args, **kwargs):
         return self.next(*args, **kwargs)
 
+    @abstractmethod
+    def _get_batches_of_transformed_samples(self, index_array):
```

Comment:
For an unimplemented method to be useful, it should have a docstring describing its specs.